src/android/plugin/google/maps/MyPluginLayout.java

```diff
@@ -281,6 +285,50 @@ public void inValidate() {
     this.frontLayer.invalidate();
   }
 
+  public void setDragging() {
+    this.mapDragging = true;
+  }
+
+  public void clearDragging() {
+    this.mapDragging = false;
+  }
+
+  public void setTouchXPosition(float x){
```

Comment:
If you need to call always both method together, it's more clear have just one public method `setTouchPosition(float x, float y)`